benchexec/tools/WitForJBMC.py

```diff
@@ -0,0 +1,32 @@
+import benchexec.tools.template
```

Comment:
Please add a header with copyright and license information in the REUSE format (cf. other files for examples), without it we cannot merge it.

If you have no further preference, we would appreciate it if you assign the copyright to Dirk Beyer and we can use Apache 2.0 as the license.

Reply:
Thanks for adding the license and the copyright!

However, we require the use of the [REUSE format](https://reuse.software/) for license headers in this project, because these are machine readable. Please update the header accordingly (just like in all other files in this repo).